index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>KMarketer</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        background-image: url(background.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+        background-attachment: fixed;
+        position: relative;
+        margin: 0 auto;
+      }
+      body::before {
+        content: "";
+        display: block;
+        width: 100%;
+        height: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: rgba(0, 0, 0, 0.5);
+        z-index: -1;
+      }
+      .container {
+        display: flex;
+        max-width: 1050px;
+        width: 860px;
+        flex-direction: column;
+        align-items: center;
+        margin: 0 auto;
+        padding: 40px 0px;
+        gap: 40px;
+      }
+      h1 {
+        width: 573px;
+        height: 143px;
+        color: #ffffff;
+        text-align: center;
+        font-family: Montserrat;
+        font-size: 58px;
+        font-style: normal;
+        font-weight: 800;
+        line-height: 80px;
+        letter-spacing: 0.2px;
+      }
+      h4 {
+        width: 539px;
+        font-size: 20px;
+        color: #ffffff;
+        text-align: center;
+        font-family: Montserrat;
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: 30px;
+        letter-spacing: 0.2px;
+      }
+      .btn1,
+      .btn2 {
+        color: #ffffff;
+        background-color: #ffa62b;
+        width: 185px;
+        padding: 10px 36px;
+        border-radius: 37px;
+        font-size: 14px;
+        font-weight: 700;
+        letter-spacing: 0.2px;
+        line-height: 28px;
+        gap: 10px;
+      }
+      .btn2 {
+        background-color: transparent;
+        width: 154px;
+        border-width: 1px;
+        border-color: #ffffff;
+      }
+      .top-banner {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+      }
+      .header {
+        display: flex;
+        max-width: 1050px;
+        justify-content: space-between;
+        align-items: center;
+        gap: 462px;
+        padding: 23px 164px 23px 136px;
```

Comment:
I bet this padding is redundant, you already set max width, it should be enough

Reply:
Thanks for the feedback! It was quite hard tbh and took me a while to make it look more or less okayish :) At some point felt like I have a mess. Anyway, an outside perspective helps a loooot. Will implement the changes right away.

Reply:
It is hard at the beginning, but the more you do the easier it becomes. So take your time and don't worry